Documentation/docs/releases/5.0.md

```diff
@@ -26,8 +26,8 @@ conda install -c conda-forge itk
 
 **Library Sources**
 
-- [InsightToolkit-5.0.0.tar.gz](https://github.com/InsightSoftwareConsortium/ITK/releases/download/v5.0.0/InsightToolkit-5.0.0.tar.gz)
-- [InsightToolkit-5.0.0.zip](https://github.com/InsightSoftwareConsortium/ITK/releases/download/v5.0.0/InsightToolkit-5.0.0.zip)
+- [ITK-5.0.0.tar.gz](https://github.com/InsightSoftwareConsortium/ITK/releases/download/v5.0.0/-5.0.0.tar.gz)
```

Comment:
This link does not work. Also, this file is an archival document, it should probably not be changed.